StudentProject/StudentData.java

```diff
@@ -0,0 +1,31 @@
+package StudentProject;
+
+class StudentData {
+    private int StudentID;
+    private String StudentFirstName;
+    private String StudentLastName;
+
+    StudentData(int StudentID, String StudentFirstName, String StudentLastName) {
```

Comment:
Why usage of a default access modifier? The Java convention for arguments/parameters is lowerCamelCase (https://www.techtarget.com/whatis/definition/lowerCamelCase).
Use _studentId_ instead of _StudentID_ ...